mage_ai/data_preparation/models/block/integration/__init__.py

```diff
@@ -314,6 +314,30 @@ class SourceBlock(IntegrationBlock):
 
 
 class DestinationBlock(IntegrationBlock):
+    def update(self, data, update_state=False):
```

Comment:
@dy46 Ok i moved the update logic to the `DestinationBlock` specifically cuz it's for updating the destination state  in particular.